.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          version: 20.10.11
+          version: docker24
```

Comment:
Maybe default ?

Reply:
I think having a fixed version is slightly better to avoid unnecessary breaks